<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EyeTracking</string>
  </property>
  <widget class="QLabel" name="source">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frames">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Element</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ISOL.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NR.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Erode</string>
   </property>
  </widget>
  <widget class="QSlider" name="frameSlider">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="firstSlider">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="morph_elem">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="morph_size">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="morph_size2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="erosion_size">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conf">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Conf.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtoutput">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="frameEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="elementEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ele1Edit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ele2Edit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="erosionEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>20</width>
     <height>711</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="radius1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="radius2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RadiusDn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RadiusUp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radius1Edit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="radius2Edit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="multipleLoad">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>180</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Multiple Videos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logs">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>180</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Multiple Logs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bigger">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="smaller">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust Bounding Box: </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
